docs/api-docs/orders/orders-api-overview.md

```diff
@@ -585,7 +585,7 @@ Accept: application/json
 
 ## Calculating totals
 
-Order `subtotal` and `total` calculate automatically; edits to the following properties trigger recalculation.
+Order `subtotal` and `total` calculate automatically; edits to the following properties trigger a recalculation.
```

Comment:
The table starting on line 602 does not render correctly. I think it should be removed as the same properties are covered in the table above (line 590).

```suggestion
Order `subtotal` and `total` calculate automatically; edits to the following properties trigger a recalculation.
```